.github/workflows/support.yml

```
@@ -11,22 +11,21 @@ jobs:
   action:
     runs-on: ubuntu-latest
     steps:
-      - uses: dessant/support-requests@v2
+      - uses: dessant/support-requests@v4
         with:
           github-token: ${{ github.token }}
           support-label: 'support'
           issue-comment: >
             👋 We use the issue tracker exclusively for bug reports and feature requests.
             However, this issue appears to be a support request. Please use our
-            [support channels](https://github.com/kivy/kivy-ios/blob/master/README.md#support)
+            [support channels](https://github.com/kivy/kivy/blob/master/README.md#support)
```

```suggestion
            [support channels](https://github.com/kivy/pyjnius/blob/master/CONTACT.md)
```

Borrowing from https://github.com/kivy/pyjnius/blob/master/.github/workflows/support.yml